features/src/main/feature/camel-features.xml

```diff
@@ -932,6 +932,7 @@
         <bundle dependency='true'>wrap:mvn:io.debezium/debezium-api/${debezium-version}$${spi-consumer}</bundle>
         <bundle dependency='true'>wrap:mvn:io.debezium/debezium-embedded/${debezium-version}</bundle>
         <bundle dependency='true'>wrap:mvn:io.debezium/debezium-core/${debezium-version}$DynamicImport-Package=io.debezium.connector.*&amp;${spi-consumer}</bundle>
+        <bundle>wrap:mvn:io.debezium/debezium-storage-file/${debezium-version}</bundle>
```

Comment:
I assume that if this is declared as a dependency, it won't be installed if needed